app/views/agent/UsingSoftware.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import services.AccountingPeriodService
+@import views.html.helpers.{ContinueButton, YesNoRadioHelper, PageHeadingHelper}
+@import views.html.templates.AgentMainTemplate
+@import views.html.helpers.ReturnDeadlineTable
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.newtablinkhelper.NewTabLinkHelper
+@import forms.agent.UsingSoftwareForm.fieldName
+
+@this(
+    mainTemplate: AgentMainTemplate,
+    yesNoRadioHelper: YesNoRadioHelper,
+    govukWarningText: GovukWarningText,
+    continueButton: ContinueButton,
+    pageHeadingHelper: PageHeadingHelper,
+    accountingPeriodService: AccountingPeriodService,
+    table: ReturnDeadlineTable,
+    form : FormWithCSRF,
+    govukDetails : GovukDetails,
+    hmrcNewTabLinkHelper: HmrcNewTabLinkHelper
+)
+
+@(usingSoftwareForm: Form[YesNo], postAction: Call, clientName: String, clientNino: String, backUrl: Option[String])(implicit request: Request[_], messages: Messages)
+
+
+@mainTemplate(title = messages("agent.using-software.heading"), optForm = Some(usingSoftwareForm), backLink = backUrl) {
+  @form(action = postAction) {
+
+    @pageHeadingHelper(
+        heading = messages("agent.using-software.heading"),
+        caption = messages("agent.using-software.caption", clientName, clientNino),
+        isSection = false
+    )
+    <p class="govuk-body">
+        @FindSoftwareNewTabLink()
+    </p>
+
+    @yesNoRadioHelper(
+      field = usingSoftwareForm(fieldName),
+      legend = messages("agent.using-software.heading"),
+      isPageHeading = false,
+      yesHint = Some(Text(messages("agent.using-software.yes-hint"))),
+      noHint = Some(Text(messages("agent.using-software.no-hint")))
+    )
+    @continueButton()
+  }
+}
+
+@FindSoftwareNewTabLink() = @{
```

Comment:
```suggestion
@findSoftwareNewTabLink() = @{
```
Function name start with lowercase